### PR TITLE
Fix: Button Overlap Issue in Responsive Design for Screen Widths Belo…

### LINKED ISSUE #1822: Button Covered by Image - UI Bug
### Description
There is a UI issue where an image is covering a button on screens below 767px  which should not be the case . Notably, this issue does not occur with any other buttons throughout the website except this section.
### Screenshots
Screenshot 2024-07-13 124603

### Expected Behavior
The button should be fully visible, not covered by any images , on screens below 767px.
Screenshot 2024-07-13 152131

### Reproduce bug
Go to the browser 
Go to inspect 
Check the website home page on screen below 767px and check get snapshots of your infrastructure directly in your PRs section.

### Enviroment:
 - OS: Windows
 - Browser: Chrome 
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by rahulrana701**
…w 767px/signoff

 Button is fully visible, not covered by any images in [Infrastructure directly in your PRs ] section at home page. , on screens below 767px.

This PR fixes #1822

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by Ashparshp — COMMENTED** [review]
Thank you for your contribution. Please avoid mixing formatting changes with functional updates, as it makes reviews harder. If you can fix this, please do so as well.

**Comment by leecalcote** [comment]
> Thank you for your contribution. Please avoid mixing formatting changes with functional updates, as it makes reviews harder. If you can fix this, please do so as well.

Thanks for this, @Ashparshp

File: _sass/features.scss [hunk 1]
```diff
@@ -1,239 +1,234 @@
 .features {
```
Comment (sudhanshutech) [inline _sass/features.scss:1]: @rahulrana701 will you please remove the unnecessary line changes or fix this lint changes. Its hard to see what is the real change.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by Ashparshp** [comment]
@sudhanshutech We're closing this PR and proceeding with another created by @amitamrutiya since we haven't heard back from @rahulrana701 for a month. Let's close this PR and move forward with the new one. Thanks, @rahulrana701.

**Comment by rahulrana701** [comment]
I apologize for the recent delay. I have been dealing with health issues for the past week and a half, and the intense placement process has kept me very busy with college commitments. I assure you that I will prioritize and complete all assigned work moving forward. This PR was not up to standard because my Git was facing some issues at that moment, which affected its quality. I am committed to delivering higher-quality work from now on. Thank you for your understanding.

**Pull request closed without merging**